src/integrationTest/java/uk/gov/hmcts/darts/common/controller/CourthouseApiTest.java

```diff
@@ -144,6 +153,39 @@ void courthousesWithRegionAndSecurityGroupsGet() throws Exception {
 
     }
 
+    @Test
+    @Transactional
+    void courthousesGetWithRegionId() throws Exception {
+        CourthouseEntity ch1 = dartsDatabase.createCourthouseUnlessExists("HAVERFORDWEST");
+
+        RegionEntity region1 = new RegionEntity();
+        region1.setRegionName("Reading");
+        regionRepository.save(region1);
+        ch1.setRegion(region1);
+
+        dartsDatabase.createCourthouseUnlessExists("SWANSEA");
+
+        CourthouseEntity ch3 = dartsDatabase.createCourthouseUnlessExists("MANCHESTER");
+        RegionEntity region3 = new RegionEntity();
+        region3.setRegionName("KNOWLE");
+        regionRepository.save(region3);
+        ch3.setRegion(region3);
+
+        MockHttpServletRequestBuilder requestBuilder = get("/courthouses")
+            .contentType(MediaType.APPLICATION_JSON_VALUE);
+        MvcResult response = mockMvc.perform(requestBuilder).andExpect(status().isOk())
+            .andExpect(jsonPath("$[0].courthouse_name", is("HAVERFORDWEST")))
+            .andExpect(jsonPath("$[0].region_id", is(notNullValue())))
```

Comment:
The test would be even more valuable if we assert that the id is the exact value that we expect, rather than simply not null. You should be able to get the id from the `RegionEntity` returned from L163.

Reply:
updated